main.go

```diff
@@ -567,18 +726,37 @@ func main() {
 			Name:   "generate-markdown",
 			Hidden: true,
 		},
-		&cli.StringFlag{
-			Name:   "log-level",
-			Hidden: true,
-			Value:  "error",
-		},
 		&cli.BoolFlag{
 			Name:    "no-color",
 			Hidden:  false,
 			Value:   false,
 			EnvVars: []string{"NO_COLOR"},
 		},
+		&cli.StringFlag{
+			Name:      "config",
+			Hidden:    true,
+			Value:     defaultConfigFilePath,
+			TakesFile: true,
+			Usage:     "Read config values from `FILE`",
+		},
+		altsrc.NewStringFlag(&cli.StringFlag{
+			Name:   cliCertFile,
```

Comment:
These CLI options are used to configure the cert and key file path. This path is read from the configuration file but I added as a hidden hidden option as it can be use for debugging.  Same idea as [yggd ](https://github.com/RedHatInsights/yggdrasil/blob/main/cmd/yggd/main.go#L57)

Reply:
`cliCertFile` is not a string; it's a `const` named `cliCertFile` with a value of `"cert-file"`. This is a convention [introduced in `yggd`](https://github.com/RedHatInsights/yggdrasil/blob/main/cmd/yggd/config.go#L12) to consolidate all CLI flag names. The end result is that the flag name is "`--cert-file`", even though the `Name:` field is given a variable rather than a string literal.

Reply:
Oh, I'm sorry. I overlooked this. BTW: Is there way how to display help with all hidden options and commands?

Reply:
Not that I'm aware of. We're using `urfave/cli`, and the last time I looked, the package didn't offer any such feature.